src/libNode/FinalBlockProcessing.cpp

```diff
@@ -708,11 +713,12 @@ bool Node::ProcessFinalBlockCore(const bytes& message, unsigned int offset,
 
   DetachedFunction(1, resumeBlackList);
 
+  if (!LoadUnavailableMicroBlockHashes(
+          txBlock, txBlock.GetHeader().GetBlockNum(), toSendTxnToLookup)) {
+    return false;
+  }
+
   if (!isVacuousEpoch) {
```

Comment:
Why bring this outside?

Reply:
vacuous also load. so as for securing the completeness of coinbase info